src/Pecee/Http/Service/Tinder.php

```diff
@@ -57,6 +57,10 @@ protected function authenticate() {
         }
     }
 
+	public function meta() {
```

Comment:
there is a new meta call

<img width="573" alt="screen shot 2016-01-01 at 23 40 51" src="https://cloud.githubusercontent.com/assets/1288106/12072497/22a0871e-b0e1-11e5-8eb0-7cf09f83cb50.png">